tests/test_text_generation_example.py

```diff
@@ -41,6 +41,7 @@
             ("bigcode/starcoder2-3b", 1, False, 261.07213776344133, True),
             ("adept/persimmon-8b-base", 4, False, 366.73968820698406, False),
             ("Qwen/Qwen1.5-7B", 4, False, 490.8621617893209, False),
+            ("Qwen/Qwen1.5-32B", 4, False, 120, False),
```

Comment:
why do we need both 7B and 32B of 1.5 if we already have 2.5?

Reply:
Because we said we support these models, we need to make sure there are no regression. We don't know what models users are using.

Reply:
one type of each is enough, we can't test all. but if you want to test on your own for each release, you can do that.